### DISCUSSION TITLE
A remote to control VLC player

### DISCUSSION THREAD

**Discussion by osc2nuke** [NativePHP/laravel#191] [Show and tell]
This is just an impression of a work in progress.
It is a remote to **send** LIVE streams to VLC Player.

As in the video, i use my personal database with the live-streams.

But you could also use the  **PUBLIC OPEN & FREE** world-wide TV-channel list, from the following Github repository:
[IPTV](https://github.com/iptv-org/iptv)

Controlling the player via VLC's build-in web interface over [VLC HTTP requests](https://wiki.videolan.org/VLC_HTTP_requests/) .
You can do much more with it, but i wanted for now just use it to send over live streams **<MRL>**

Sharing it via a video for inspiration purposes and show-off.
I think it is always nice for the creators of NativePHP ( @simonhamp  @mpociot ) to see what users do with it.

[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.08.23-14_13_35.webm](https://github.com/NativePHP/laravel/assets/5169035/8b454fda-278c-476f-8490-a0207c62b720)

**Comment by eskayamadeus** [comment]
Awesome work!

**Comment by simonhamp** [comment]
I'm only just seeing this now! This is very cool... have you made much progress with it since?
